zerver/openapi/zulip.yaml

```diff
@@ -10264,6 +10264,8 @@ paths:
             description: |
               You may pass in one or more of the parameters mentioned above
               as URL query parameters, like so:
+      x-parameter-description: |
+        **Note**: The following parameters are all URL query parameters.
```

Comment:
I think we should just delete this one; all HTTP GET requests are supposed to be encoded this way, and we cover this in the examples I think.

Reply:
It indeed feels little redundant -- removed this description.